corehq/ex-submodules/casexml/apps/phone/tests/utils.py

```diff
@@ -64,7 +64,15 @@ def deprecated_generate_restore_payload(project, user, restore_id="", version=V1
     ).get_payload().as_string()
 
 
-def call_fixture_generator(gen, restore_user, project=None, last_sync=None, app=None, device_id=''):
+def call_fixture_generator(
+    gen,
+    restore_user,
+    project=None,
+    last_sync=None,
+    app=None,
+    device_id='',
+    current_sync_log=None,
```

Comment:
This new param was includes in this test util as the v2 report fixture access the `current_sync_log`